### PR TITLE
Debug frontend

### LINKED ISSUE #601: @weex/plugins/debug npm run build 异常
操作系统 mac
node v10.17.0
npm 6.11.3

error message:

> ERROR in ./src/views/analyze/analyze.ts
> Module not found: Error: Can't resolve 'bootstrap-vue/es/components/button/button' in '/Users/cnryb/Documents/GitHub/weex-cli/packages/@weex/plugins/debug/frontend/src/views/analyze'
>  @ ./src/views/analyze/analyze.ts 43:0-64
>  @ ./src/views/analyze/index.ts
>  @ ./src/router.ts
>  @ ./src/main.ts
> 
> ERROR in ./src/views/eval/eval.ts
> Module not found: Error: Can't resolve 'bootstrap-vue/es/components/button/button' in '/Users/cnryb/Documents/GitHub/weex-cli/packages/@weex/plugins/debug/frontend/src/views/eval'
>  @ ./src/views/eval/eval.ts 38:0-64
>  @ ./src/views/eval/index.ts
>  @ ./src/router.ts
>  @ ./src/main.ts
> 
> ERROR in ./src/views/weex/weex.ts
> Module not found: Error: Can't resolve 'bootstrap-vue/es/components/button/button' in '/Users/cnryb/Documents/GitHub/weex-cli/packages/@weex/plugins/debug/frontend/src/views/weex'
>  @ ./src/views/weex/weex.ts 50:0-64
>  @ ./src/views/weex/index.ts
>  @ ./src/router.ts
>  @ ./src/main.ts
> 
> ERROR in ./src/components/navbar/navbar.ts
> Module not found: Error: Can't resolve 'bootstrap-vue/es/components/collapse/collapse' in '/Users/cnryb/Documents/GitHub/weex-cli/packages/@weex/plugins/debug/frontend/src/components/navbar'
>  @ ./src/components/navbar/navbar.ts 38:0-70
>  @ ./src/components/navbar/index.ts
>  @ ./src/views/client/client.ts
>  @ ./src/views/client/index.ts
>  @ ./src/router.ts
>  @ ./src/main.ts
> 
> ERROR in ./src/views/analyze/analyze.ts
> Module not found: Error: Can't resolve 'bootstrap-vue/es/components/embed/embed' in '/Users/cnryb/Documents/GitHub/weex-cli/packages/@weex/plugins/debug/frontend/src/views/analyze'
>  @ ./src/views/analyze/analyze.ts 40:0-61
>  @ ./src/views/analyze/index.ts
>  @ ./src/router.ts
>  @ ./src/main.ts
> 
> ERROR in ./src/views/eval/eval.ts
> Module not found: Error: Can't resolve 'bootstrap-vue/es/components/embed/embed' in '/Users/cnryb/Documents/GitHub/weex-cli/packages/@weex/plugins/debug/frontend/src/views/eval'
>  @ ./src/views/eval/eval.ts 36:0-61
>  @ ./src/views/eval/index.ts
>  @ ./src/router.ts
>  @ ./src/main.ts
> 
> ERROR in ./src/views/weex/weex.ts
> Module not found: Error: Can't resolve 'bootstrap-vue/es/components/embed/embed' in '/Users/cnryb/Documents/GitHub/weex-cli/packages/@weex/plugins/debug/frontend/src/views/weex'
>  @ ./src/views/weex/weex.ts 48:0-61
>  @ ./src/views/weex/index.ts
>  @ ./src/router.ts
>  @ ./src/main.ts
> 
> ERROR in ./src/views/analyze/analyze.ts
> Module not found: Error: Can't resolve 'bootstrap-vue/es/components/form-checkbox/form-checkbox' in '/Users/cnryb/Documents/GitHub/weex-cli/packages/@weex/plugins/debug/frontend/src/views/analyze'
>  @ ./src/views/analyze/analyze.ts 39:0-80
>  @ ./src/views/analyze/index.ts
>  @ ./src/router.ts
>  @ ./src/main.ts
> 
> ERROR in ./src/views/analyze/analyze.ts
> Module not found: Error: Can't resolve 'bootstrap-vue/es/components/form-radio/form-radio-group' in '/Users/cnryb/Documents/GitHub/weex-cli/packages/@weex/plugins/debug/frontend/src/views/analyze'
>  @ ./src/views/analyze/analyze.ts 50:0-86
>  @ ./src/views/analyze/index.ts
>  @ ./src/router.ts
>  @ ./src/main.ts
> 
> ERROR in ./src/views/eval/eval.ts
> Module not found: Error: Can't resolve 'bootstrap-vue/es/components/form-select/form-select' in '/Users/cnryb/Documents/GitHub/weex-cli/packages/@weex/plugins/debug/frontend/src/views/eval'
>  @ ./src/views/eval/eval.ts 37:0-78
>  @ ./src/views/eval/index.ts
>  @ ./src/router.ts
>  @ ./src/main.ts
> 
> ERROR in ./src/views/weex/weex.ts
> Module not found: Error: Can't resolve 'bootstrap-vue/es/components/form-select/form-select' in '/Users/cnryb/Documents/GitHub/weex-cli/packages/@weex/plugins/debug/frontend/src/views/weex'
>  @ ./src/views/weex/weex.ts 49:0-78
>  @ ./src/views/weex/index.ts
>  @ ./src/router.ts
>  @ ./src/main.ts
> 
> ERROR in ./src/views/default/default.ts
> Module not found: Error: Can't resolve 'bootstrap-vue/es/components/layout/col' in '/Users/cnryb/Documents/GitHub/weex-cli/packages/@weex/plugins/debug/frontend/src/views/default'
>  @ ./src/views/default/default.ts 36:0-58
>  @ ./src/views/default/index.ts
>  @ ./src/router.ts
>  @ ./src/main.ts
> 
> ERROR in ./src/views/home/home.ts
> Module not found: Error: Can't resolve 'bootstrap-vue/es/components/layout/col' in '/Users/cnryb/Documents/GitHub/weex-cli/packages/@weex/plugins/debug/frontend/src/views/home'
>  @ ./src/views/home/home.ts 76:0-58
>  @ ./src/views/home/index.ts
>  @ ./src/router.ts
>  @ ./src/main.ts
> 
> ERROR in ./src/views/default/default.ts
> Module not found: Error: Can't resolve 'bootstrap-vue/es/components/layout/container' in '/Users/cnryb/Documents/GitHub/weex-cli/packages/@weex/plugins/debug/frontend/src/views/default'
>  @ ./src/views/default/default.ts 35:0-70
>  @ ./src/views/default/index.ts
>  @ ./src/router.ts
>  @ ./src/main.ts
> 
> ERROR in ./src/views/home/home.ts
> Module not found: Error: Can't resolve 'bootstrap-vue/es/components/layout/container' in '/Users/cnryb/Documents/GitHub/weex-cli/packages/@weex/plugins/debug/frontend/src/views/home'
>  @ ./src/views/home/home.ts 75:0-70
>  @ ./src/views/home/index.ts
>  @ ./src/router.ts
>  @ ./src/main.ts
> 
> ERROR in ./src/views/default/default.ts
> Module not found: Error: Can't resolve 'bootstrap-vue/es/components/layout/row' in '/Users/cnryb/Documents/GitHub/weex-cli/packages/@weex/plugins/debug/frontend/src/views/default'
>  @ ./src/views/default/default.ts 37:0-58
>  @ ./src/views/default/index.ts
>  @ ./src/router.ts
>  @ ./src/main.ts
> 
> ERROR in ./src/views/home/home.ts
> Module not found: Error: Can't resolve 'bootstrap-vue/es/components/layout/row' in '/Users/cnryb/Documents/GitHub/weex-cli/packages/@weex/plugins/debug/frontend/src/views/home'
>  @ ./src/views/home/home.ts 77:0-58
>  @ ./src/views/home/index.ts
>  @ ./src/router.ts
>  @ ./src/main.ts
> 
> ERROR in ./src/views/analyze/analyze.ts
> Module not found: Error: Can't resolve 'bootstrap-vue/es/components/modal/modal' in '/Users/cnryb/Documents/GitHub/weex-cli/packages/@weex/plugins/debug/frontend/src/views/analyze'
>  @ ./src/views/analyze/analyze.ts 44:0-61
>  @ ./src/views/analyze/index.ts
>  @ ./src/router.ts
>  @ ./src/main.ts
> 
> ERROR in ./src/views/weex/weex.ts
> Module not found: Error: Can't resolve 'bootstrap-vue/es/components/modal/modal' in '/Users/cnryb/Documents/GitHub/weex-cli/packages/@weex/plugins/debug/frontend/src/views/weex'
>  @ ./src/views/weex/weex.ts 52:0-61
>  @ ./src/views/weex/index.ts
>  @ ./src/router.ts
>  @ ./src/main.ts
> 
> ERROR in ./src/components/navbar/navbar.ts
> Module not found: Error: Can't resolve 'bootstrap-vue/es/components/nav/nav-item' in '/Users/cnryb/Documents/GitHub/weex-cli/packages/@weex/plugins/debug/frontend/src/components/navbar'
>  @ ./src/components/navbar/navbar.ts 39:0-64
>  @ ./src/components/navbar/index.ts
>  @ ./src/views/client/client.ts
>  @ ./src/views/client/index.ts
>  @ ./src/router.ts
>  @ ./src/main.ts
> 
> ERROR in ./src/components/navbar/navbar.ts
> Module not found: Error: Can't resolve 'bootstrap-vue/es/components/navbar/navbar' in '/Users/cnryb/Documents/GitHub/weex-cli/packages/@weex/plugins/debug/frontend/src/components/navbar'
>  @ ./src/components/navbar/navbar.ts 40:0-64
>  @ ./src/components/navbar/index.ts
>  @ ./src/views/client/client.ts
>  @ ./src/views/client/index.ts
>  @ ./src/router.ts
>  @ ./src/main.ts
> 
> ERROR in ./src/components/navbar/navbar.ts
> Module not found: Error: Can't resolve 'bootstrap-vue/es/components/navbar/navbar-brand' in '/Users/cnryb/Documents/GitHub/weex-cli/packages/@weex/plugins/debug/frontend/src/components/navbar'
>  @ ./src/components/navbar/navbar.ts 42:0-75
>  @ ./src/components/navbar/index.ts
>  @ ./src/views/client/client.ts
>  @ ./src/views/client/index.ts
>  @ ./src/router.ts
>  @ ./src/main.ts
> 
> ERROR in ./src/components/navbar/navbar.ts
> Module not found: Error: Can't resolve 'bootstrap-vue/es/components/navbar/navbar-nav' in '/Users/cnryb/Documents/GitHub/weex-cli/packages/@weex/plugins/debug/frontend/src/components/navbar'
>  @ ./src/components/navbar/navbar.ts 43:0-71
>  @ ./src/components/navbar/index.ts
>  @ ./src/views/client/client.ts
>  @ ./src/views/client/index.ts
>  @ ./src/router.ts
>  @ ./src/main.ts
> 
> ERROR in ./src/components/navbar/navbar.ts
> Module not found: Error: Can't resolve 'bootstrap-vue/es/components/navbar/navbar-toggle' in '/Users/cnryb/Documents/GitHub/weex-cli/packages/@weex/plugins/debug/frontend/src/components/navbar'
>  @ ./src/components/navbar/navbar.ts 41:0-77
>  @ ./src/components/navbar/index.ts
>  @ ./src/views/client/client.ts
>  @ ./src/views/client/index.ts
>  @ ./src/router.ts
>  @ ./src/main.ts
> 
> ERROR in ./src/views/analyze/analyze.ts
> Module not found: Error: Can't resolve 'bootstrap-vue/es/components/popover/popover' in '/Users/cnryb/Documents/GitHub/weex-cli/packages/@weex/plugins/debug/frontend/src/views/analyze'
>  @ ./src/views/analyze/analyze.ts 51:0-67
>  @ ./src/views/analyze/index.ts
>  @ ./src/router.ts
>  @ ./src/main.ts
> 
> ERROR in ./src/views/home/home.ts
> Module not found: Error: Can't resolve 'bootstrap-vue/es/components/popover/popover' in '/Users/cnryb/Documents/GitHub/weex-cli/packages/@weex/plugins/debug/frontend/src/views/home'
>  @ ./src/views/home/home.ts 79:0-67
>  @ ./src/views/home/index.ts
>  @ ./src/router.ts
>  @ ./src/main.ts
> 
> ERROR in ./src/views/analyze/analyze.ts
> Module not found: Error: Can't resolve 'bootstrap-vue/es/components/progress/progress' in '/Users/cnryb/Documents/GitHub/weex-cli/packages/@weex/plugins/debug/frontend/src/views/analyze'
>  @ ./src/views/analyze/analyze.ts 42:0-70
>  @ ./src/views/analyze/index.ts
>  @ ./src/router.ts
>  @ ./src/main.ts
> 
> ERROR in ./src/views/analyze/analyze.ts
> Module not found: Error: Can't resolve 'bootstrap-vue/es/components/table/table' in '/Users/cnryb/Documents/GitHub/weex-cli/packages/@weex/plugins/debug/frontend/src/views/analyze'
>  @ ./src/views/analyze/analyze.ts 41:0-61
>  @ ./src/views/analyze/index.ts
>  @ ./src/router.ts
>  @ ./src/main.ts
> 
> ERROR in ./src/main.ts
> Module not found: Error: Can't resolve 'bootstrap-vue/es/directives/popover/popover' in '/Users/cnryb/Documents/GitHub/weex-cli/packages/@weex/plugins/debug/frontend/src'
>  @ ./src/main.ts 30:0-68
> 
> ERROR in ./src/main.ts
> Module not found: Error: Can't resolve 'bootstrap-vue/es/directives/tooltip/tooltip' in '/Users/cnryb/Documents/GitHub/weex-cli/packages/@weex/plugins/debug/frontend/src'
>  @ ./src/main.ts 29:0-68
> 
> ERROR in [at-loader] ./node_modules/vuex/types/helpers.d.ts:5:45 
>     TS1005: ';' expected.
> 
> ERROR in [at-loader] ./node_modules/vuex/types/helpers.d.ts:5:79 
>     TS1005: ';' expected.
> 
> ERROR in [at-loader] ./node_modules/vuex/types/helpers.d.ts:8:67 
>     TS1005: ';' expected.
> 
> ERROR in [at-loader] ./node_modules/vuex/types/helpers.d.ts:8:100 
>     TS1005: ',' expected.
> 
> ERROR in [at-loader] ./node_modules/vuex/types/helpers.d.ts:8:115 
>     TS1005: ';' expected.
> 
> ERROR in [at-loader] ./node_modules/vuex/types/helpers.d.ts:5:73 
>     TS2304: Cannot find name 'infer'.
> 
> ERROR in [at-loader] ./node_modules/vuex/types/helpers.d.ts:5:79 
>     TS2304: Cannot find name 'R'.
> 
> ERROR in [at-loader] ./node_modules/vuex/types/helpers.d.ts:5:89 
>     TS2304: Cannot find name 'R'.
> 
> ERROR in [at-loader] ./node_modules/vuex/types/helpers.d.ts:5:93 
>     TS2693: 'never' only refers to a type, but is being used as a value here.
> 
> ERROR in [at-loader] ./node_modules/vuex/types/helpers.d.ts:8:85 
>     TS2370: A rest parameter must be of an array type.
> 
> ERROR in [at-loader] ./node_modules/vuex/types/helpers.d.ts:8:94 
>     TS2304: Cannot find name 'infer'.
> 
> ERROR in [at-loader] ./node_modules/vuex/types/helpers.d.ts:8:109 
>     TS2304: Cannot find name 'infer'.
> 
> ERROR in [at-loader] ./node_modules/vuex/types/helpers.d.ts:8:115 
>     TS2304: Cannot find name 'R'.
> 
> ERROR in [at-loader] ./node_modules/vuex/types/helpers.d.ts:8:120 
>     TS2370: A rest parameter must be of an array type.
> 
> ERROR in [at-loader] ./node_modules/vuex/types/helpers.d.ts:8:129 
>     TS2304: Cannot find name 'Args'.
> 
> ERROR in [at-loader] ./node_modules/vuex/types/helpers.d.ts:8:138 
>     TS2304: Cannot find name 'R'.
> 
> ERROR in [at-loader] ./node_modules/vuex/types/helpers.d.ts:8:142 
>     TS2693: 'never' only refers to a type, but is being used as a value here.
> 
> ERROR in [at-loader] ./src/components/navbar/navbar.ts:19:23 
>     TS2307: Cannot find module 'bootstrap-vue/es/components/collapse/collapse'.
> 
> ERROR in [at-loader] ./src/components/navbar/navbar.ts:20:22 
>     TS2307: Cannot find module 'bootstrap-vue/es/components/nav/nav-item'.
> 
> ERROR in [at-loader] ./src/components/navbar/navbar.ts:21:21 
>     TS2307: Cannot find module 'bootstrap-vue/es/components/navbar/navbar'.
> 
> ERROR in [at-loader] ./src/components/navbar/navbar.ts:22:27 
>     TS2307: Cannot find module 'bootstrap-vue/es/components/navbar/navbar-toggle'.
> 
> ERROR in [at-loader] ./src/components/navbar/navbar.ts:23:26 
>     TS2307: Cannot find module 'bootstrap-vue/es/components/navbar/navbar-brand'.
> 
> ERROR in [at-loader] ./src/components/navbar/navbar.ts:24:24 
>     TS2307: Cannot find module 'bootstrap-vue/es/components/navbar/navbar-nav'.
> 
> ERROR in [at-loader] ./src/main.ts:29:23 
>     TS2307: Cannot find module 'bootstrap-vue/es/directives/tooltip/tooltip'.
> 
> ERROR in [at-loader] ./src/main.ts:30:23 
>     TS2307: Cannot find module 'bootstrap-vue/es/directives/popover/popover'.
> 
> ERROR in [at-loader] ./src/views/analyze/analyze.ts:24:23 
>     TS2307: Cannot find module 'bootstrap-vue/es/components/form-checkbox/form-checkbox'.
> 
> ERROR in [at-loader] ./src/views/analyze/analyze.ts:25:20 
>     TS2307: Cannot find module 'bootstrap-vue/es/components/embed/embed'.
> 
> ERROR in [at-loader] ./src/views/analyze/analyze.ts:26:20 
>     TS2307: Cannot find module 'bootstrap-vue/es/components/table/table'.
> 
> ERROR in [at-loader] ./src/views/analyze/analyze.ts:27:23 
>     TS2307: Cannot find module 'bootstrap-vue/es/components/progress/progress'.
> 
> ERROR in [at-loader] ./src/views/analyze/analyze.ts:28:21 
>     TS2307: Cannot find module 'bootstrap-vue/es/components/button/button'.
> 
> ERROR in [at-loader] ./src/views/analyze/analyze.ts:29:20 
>     TS2307: Cannot find module 'bootstrap-vue/es/components/modal/modal'.
> 
> ERROR in [at-loader] ./src/views/analyze/analyze.ts:35:29 
>     TS2307: Cannot find module 'bootstrap-vue/es/components/form-radio/form-radio-group'.
> 
> ERROR in [at-loader] ./src/views/analyze/analyze.ts:36:22 
>     TS2307: Cannot find module 'bootstrap-vue/es/components/popover/popover'.
> 
> ERROR in [at-loader] ./src/views/default/default.ts:25:24 
>     TS2307: Cannot find module 'bootstrap-vue/es/components/layout/container'.
> 
> ERROR in [at-loader] ./src/views/default/default.ts:26:18 
>     TS2307: Cannot find module 'bootstrap-vue/es/components/layout/col'.
> 
> ERROR in [at-loader] ./src/views/default/default.ts:27:18 
>     TS2307: Cannot find module 'bootstrap-vue/es/components/layout/row'.
> 
> ERROR in [at-loader] ./src/views/eval/eval.ts:24:20 
>     TS2307: Cannot find module 'bootstrap-vue/es/components/embed/embed'.
> 
> ERROR in [at-loader] ./src/views/eval/eval.ts:25:25 
>     TS2307: Cannot find module 'bootstrap-vue/es/components/form-select/form-select'.
> 
> ERROR in [at-loader] ./src/views/eval/eval.ts:26:21 
>     TS2307: Cannot find module 'bootstrap-vue/es/components/button/button'.
> 
> ERROR in [at-loader] ./src/views/home/home.ts:25:24 
>     TS2307: Cannot find module 'bootstrap-vue/es/components/layout/container'.
> 
> ERROR in [at-loader] ./src/views/home/home.ts:26:18 
>     TS2307: Cannot find module 'bootstrap-vue/es/components/layout/col'.
> 
> ERROR in [at-loader] ./src/views/home/home.ts:27:18 
>     TS2307: Cannot find module 'bootstrap-vue/es/components/layout/row'.
> 
> ERROR in [at-loader] ./src/views/home/home.ts:29:22 
>     TS2307: Cannot find module 'bootstrap-vue/es/components/popover/popover'.
> 
> ERROR in [at-loader] ./src/views/weex/weex.ts:25:20 
>     TS2307: Cannot find module 'bootstrap-vue/es/components/embed/embed'.
> 
> ERROR in [at-loader] ./src/views/weex/weex.ts:26:25 
>     TS2307: Cannot find module 'bootstrap-vue/es/components/form-select/form-select'.
> 
> ERROR in [at-loader] ./src/views/weex/weex.ts:27:21 
>     TS2307: Cannot find module 'bootstrap-vue/es/components/button/button'.
> 
> ERROR in [at-loader] ./src/views/weex/weex.ts:29:20 
>     TS2307: Cannot find module 'bootstrap-vue/es/components/modal/modal'.

### PR DESCRIPTION
**Pull request by cnryb**
fix #601

### CODE REVIEW

**Review by YorkShen — COMMENTED** [review]
Thanks for your contributing, that's really a great.

IMO, you may consider revert code format change, like adding space in the following files:
* navbar.ts
* analyze.ts
* home.html
* home.ts
* weex.ts

**Comment by YorkShen** [comment]
This's really an amazing work, thanks a lot for your contribution. @cnryb

File: packages/@weex/plugins/debug/frontend/src/views/analyze/analyze.ts [hunk 1]
```diff
@@ -153,15 +153,15 @@ export class AnalyzeComponent extends Vue {
   historyChange () {
     let downloadContent
     let len = this.historys.length
-    for (let i = 0; i < len;i++) {
```
Comment (YorkShen) [inline packages/@weex/plugins/debug/frontend/src/views/analyze/analyze.ts:156]: There are still some changes about code format in `analyze.ts`, you may consider revert it.

Comment (cnryb) [reply packages/@weex/plugins/debug/frontend/src/views/analyze/analyze.ts:156]: After the first`; `there is a space after it. The second`; `there is no after.   
Maybe it was a problem.

Comment (YorkShen) [reply packages/@weex/plugins/debug/frontend/src/views/analyze/analyze.ts:156]: Ok, your idea makes sense.

**Review by YorkShen — APPROVED** [review]
LGTM

*FYI: If you are going to publish this change to npm, you need to go through Apache release process. If you are going to compile it from source and use it, then everything is fine.*

**Comment by cnryb** [comment]
@YorkShen I think it is necessary to understand the release process. Now, I want publish it to npm, what should I do？

**Comment by YorkShen** [comment]
We have a [release procedure](https://cwiki.apache.org/confluence/display/WEEX/Release+Procedure) for [incubator-weex](https://github.com/apache/incubator-weex), maybe you could take it as an example. @cnryb

**Pull request merged**